conversions/infix_to_postfix.c

```diff
@@ -0,0 +1,202 @@
+/**
+ * @file
+ * @brief Infix to Postfix Expression Conversion
```

Comment:
Provide a Wikipedia link in Markdown format.
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂

Reply:
Added that at line 6